docs/add-ons/tutorial.md

```diff
@@ -50,7 +50,7 @@ name: "Hello world"
 description: "My first real add-on!"
 version: "1.0.0"
 slug: "hello_world"
-init: false
+init: true
```

Comment:
This change is not correct, the example uses our base images, which have their own init setup (S6).

Closing this PR for that reason.

../Frenck

Reply:
should I then put the information somewhere in the docs maybe ?